app/data_access/control_data.py

```diff
@@ -330,3 +351,20 @@ def resolve_observed_infractions(self, info):
             i for i in observed_infractions if i is not None
         ]
         return observed_infractions
+
+    def resolve_can_take_pictures(self, info):
+        if self.control_type != ControlType.lic_papier:
+            return False
+
+        return self.creation_time.date() == datetime.datetime.now().date()
+
+    def resolve_pictures(self, info):
+        pictures = S3Client.list_pictures_for_control(self.id)
+        urls = [
+            S3Client.generate_presigned_url_for_picture(picture)
+            for picture in pictures
+        ]
+        return [ControlPictureOutput(url=url) for url in urls]
+
+    def resolve_pictures_expiry_date(self, info):
+        return self.creation_time.date() + timedelta(days=90)
```

Comment:
suggestion(non-blocking): use an env var for this magic number (90)

or

suggestion(if-minor): if we can get the expiry date from the s3, we will ensure we have the correct expiry date (in case we change it in the scaleway admin ; we have to change it here too)